guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,675 @@
+{
```

Comment:
<p>As we see above, the page and site were cloned successfully.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1449/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>